src/resources/map/MapFont.java

```diff
@@ -0,0 +1,29 @@
+package resources.map;
```

Comment:
I would definitely open a new enhancement issue, referencing this comment in the process. 
![image](https://github.com/JasonBarahan/TrackMyTransit/assets/122300705/3d01884c-8fe2-4358-9304-92f6287e57c5)

Since you pointed it out, you do the honours :)